addons/website_blog/views/website_blog_posts_loop.xml

```diff
@@ -98,7 +98,7 @@ according to the enabled options.
             <!-- Loop through posts: exclude the first one if already displayed as top banner -->
             <t t-foreach="posts" t-as="blog_post">
                 <!-- Assign 'colWidth': 'col-12' is default for List-View and mobile -->
-                <div t-attf-class="pb-4 col-12 #{colWidth}" t-cache="blog_post,opt_blog_list_view,opt_blog_readable,active_tag_ids">
+                <div t-attf-class="pb-4 col-12 #{colWidth}" t-cache="blog_post,opt_blog_list_view,opt_blog_readable,active_tag_ids,str(blog_post.tag_ids.ids)">
```

Comment:
If you use `blog_post.tag_ids` in t-cache instead of the str ids, the t-cache will use the write_date. You don't need to clear the cache manually.